printing/printer.ml

```diff
@@ -915,9 +915,9 @@ let pr_assumptionset env sigma s =
       | Constant kn ->
           safe_pr_constant env kn ++ safe_pr_ltype typ
       | Positive m ->
-          hov 2 (MutInd.print m ++ spc () ++ strbrk"is positive.")
+          hov 2 (MutInd.print m ++ spc () ++ strbrk"is assumed to be positive")
       | Guarded kn ->
-          hov 2 (safe_pr_constant env kn ++ spc () ++ strbrk"is positive.")
+          hov 2 (safe_pr_constant env kn ++ spc () ++ strbrk"is assumed to be guarded")
```

Comment:
If this is not affecting any output test, could you add one? I'd like to understand in which situation these messages are seen.